packages/gp2-components/src/organisms/ProjectSummaryFooter.tsx

```diff
@@ -24,15 +24,18 @@ const ProjectSummaryFooter: React.FC<ProjectSummaryFooterProps> = ({
     </IconWithLabel>
     <IconWithLabel icon={dateIcon}>
       <span>
-        {`${format(new Date(startDate), 'MMM yyyy')}${
-          endDate ? ` - ${format(new Date(endDate), 'MMM yyyy')} · ` : ''
-        }`}
-        {endDate && (
-          <span css={{ color: colors.neutral800.rgba }}>
-            {`(${formatDistance(new Date(startDate), new Date(endDate), {
-              unit: 'month',
-            })})`}
-          </span>
+        {!startDate && 'TBD'}
+        {startDate && !endDate && `${format(new Date(startDate), 'MMM yyyy')}`}
+        {startDate && endDate && (
+          <>
+            {format(new Date(startDate), 'MMM yyyy')} -{' '}
+            {format(new Date(endDate), 'MMM yyyy')} ·{' '}
+            <span css={{ color: colors.neutral800.rgba }}>
+              {`(${formatDistance(new Date(startDate), new Date(endDate), {
+                unit: 'month',
+              })})`}
+            </span>
+          </>
         )}
```

Comment:
I'm sure this could be refactored to make it a little more readable.

Maybe:

```suggestion
        
        {startDate ? `${format(new Date(startDate), 'MMM yyyy')}` : 'TBD'}
        {(startDate && endDate) ? (
          <>
            {'- '}{format(new Date(endDate), 'MMM yyyy')} ·{' '}
            <span css={{ color: colors.neutral800.rgba }}>
              {`(${formatDistance(new Date(startDate), new Date(endDate), {
                unit: 'month',
              })})`}
            </span>
          </>
        ) : null}
```